cmd/server/config.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/kubernetes-incubator/cri-o/server"
+	"github.com/opencontainers/runc/libcontainer/selinux"
+	"github.com/urfave/cli"
+)
+
+const (
+	ocidRoot   = "/var/lib/ocid"
+	conmonPath = "/usr/libexec/ocid/conmon"
+	pausePath  = "/usr/libexec/ocid/pause"
+)
+
+var commentedConfigTemplate = template.Must(template.New("config").Parse(`
+# Commented version of ocid configuration file. Generated automatically.
+
+# The "ocid" table contains all of the server options. There are no
+# bare options. "ocid" contains global options that don't really
+# relate to any particular sub-table.
+[ocid]
+
+# root is a path to the "root directory" where all information not
+# explicitly handled by other options will be stored.
+root = "{{ .Root }}"
```

Comment:
I'm actually not too sure what `config.Root` _actually_ does. Maybe @mrunalp or @runcom can help explain it to me so we can document it?